bin/fix-ci.php

```diff
@@ -0,0 +1,237 @@
+<?php
```

Comment:
Can we make this file strict and run the PhpCsFixer once?

Reply:
No, as I will extract this one into a completely separate tool 😂